contracts/Killable.sol

```diff
@@ -0,0 +1,42 @@
+pragma solidity 0.5.10;
```

Comment:
hmm, you really over-complicated this; I meant that you add the following to `Stoppable` contract:
1. State variable: `killed`
2. Modifier: `whenAlive()`
3. Function: `kill()` which can be run only if the contract was paused first.
4. Then you apply `whenAlive()` and `whenRunning()` modifiers to your Remittance functions 🙂

Reply:
I got your point. However, from my pov the stoppable contract should not be able to kill the contract. It's not its purpose. Upstream, I would like to give the opportunity to ongoing transfer to be withdrawed by the exchange shop, to prevent any lost of funds.